backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/s3/S3Client.java

```diff
@@ -112,7 +112,17 @@ private void initializeWithTemporaryCredentials(Regions awsRegion)
     AWSSecurityTokenService stsClient = null;
     AmazonS3 newS3Client = null;
     try {
-      stsClient = AWSSecurityTokenServiceClientBuilder.standard().withRegion(awsRegion).build();
+      final var stsClientBuilder =
+          AWSSecurityTokenServiceClientBuilder.standard().withRegion(awsRegion);
+      if (!CommonUtils.appendOptionalTelepresencePath("foo").equals("foo")) {
```

Comment:
"foo"? Okay, I get the intent. I'd rather see helper in `CommonUtils` for this, but in the meanwhile would empty string also work here?   Having the specific value `"foo"` made me wonder for a second if I was looking at test.  :smile:

```suggestion
      if (String.isEmpty(CommonUtils.appendOptionalTelepresencePath("")) {
```

Reply:
Wait, we're in modeldb. Let's update `CommonUtils`.

Reply:
Holding off on making this change because the plan is to combine this with the Registry change into common code, so leaving it as similar to that as possible won't cause an issue.

Reply:
Discussed offline and approved code as direct duplicate other code.